### PR TITLE
Added Admin Services SDK

### PR DESCRIPTION
**Pull request by NickXitco**
## PR summary
Added Admin Services SDK

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [X] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features) (API docs in progress)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [X] No

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/IBM/go-sdk-core/v5 v5.4.4
+	github.com/IBM/platform-services-go-sdk v0.18.18 // indirect
```
Comment (ashishth09) [inline go.mod:7]: We shouldn't depend on platform-services-go-sdk. I see you used it for composing the headers, please use github.com/IBM/scc-go-sdk/common instead.

**Review by ashishth09 — CHANGES_REQUESTED** [review]

**Comment by ashishth09** [comment]
@NickXitco please run 
```
go fmt ./...
```
from the root project

**Comment by gary1998** [comment]
Couple of changes are required:
- Need to replace the import statement on `https://github.com/NickXitco/scc-go-sdk/blob/admin_services_sdk/adminserviceapiv1/admin_service_api_v1.go#L33` to `common "github.com/IBM/scc-go-sdk/common"`
- Once the above change is placed, do run `go mod tidy` to update the `go.mod` and `go.sum`
- Another change is to add `echo "${ADMIN_SERVICE_ENV}" | base64 -d >> admin_services_v1.env` to `build/testScript.sh` before `make test-int` command.

**Pull request merged**